app/assets/javascripts/utils/media_wiki_references_utils.js

```diff
@@ -4,6 +4,10 @@ import { queryUrl } from './article_finder_utils';
 import { chunk } from 'lodash-es';
 import { getReferencesCount } from './revision_utils';
 
+import promiseLimit from 'promise-limit';
+
+const limit = promiseLimit(10);
```

Comment:
Did you test this? 10 might be too many.

Reply:
It did work for me, although one request did fail the second time I ran it. Do I just set it to 5?

Reply:
yeah, I think 5 is probably good.